judgels-frontends/raphael/src/routes/contests/contests/single/components/ContestAnnouncementsWidget/ContestAnnouncementsWidget.jsx

```diff
@@ -15,12 +17,16 @@ class ContestAnnouncementsWidget extends React.Component {
 
   componentDidUpdate(prevProps) {
     if (this.props.announcementCount > prevProps.announcementCount) {
-      this.props.onAlertNewAnnouncements();
+      // TODO(lungsin): change the notification tag to be more proper, e.g. using announcement JID.
+      const timestamp = Math.floor(Date.now() / SingleContestDataRoute.GET_CONFIG_TIMEOUT); // Use timestamp for notification tag
```

Comment:
Can we instead add the constant in `src/modules/api/uriel/contestWeb.js` and have all 3 files refer to the const